makefile

```diff
@@ -73,11 +79,18 @@ medgen-disease-extract.owl: medgen-disease-extract.obo
 	owltools $< -o $@
 
 # SSSOM ----------------------------------
-medgen.obographs.json:
-	robot convert -i medgen-disease-extract.owl -o $@
-
-medgen.sssom.tsv: medgen.obographs.json
-	sssom parse medgen.obographs.json -I obographs-json -m config/medgen.sssom-metadata.yml -o $@
+# todo: comemented out old pipeline: remove
+#medgen.obographs.json:
+#	robot convert -i medgen-disease-extract.owl -o $@
+#
+#medgen.sssom.tsv: medgen.obographs.json
+#	sssom parse medgen.obographs.json -I obographs-json -m config/medgen.sssom-metadata.yml -o $@
+sssom: umls-hpo.sssom.tsv
+	sssom validate umls-hpo.sssom.tsv
```

Comment:
## SSSOM validation
As with the [ICD11 PR](https://github.com/monarch-initiative/icd11/pull/12#discussion_r1554726972), I also added SSSOM validation here.